test/unit/dropbox.js

```diff
@@ -31,6 +39,40 @@ describe('Dropbox', () => {
       chai.assert.deepEqual({}, dbx.rpcRequest.getCall(0).args[1]);
     });
 
+    it('calls the correct request method (Download)', () => {
+      const dbx = new Dropbox();
+      const rpcSpy = sinon.spy(dbx, 'downloadRequest');
```

Comment:
Should the naming of this be `downloadSpy` since it is a download request?

Reply:
Yea that was a copy error lol